spec/integration/spec_file_load_errors_spec.rb

```diff
@@ -82,6 +82,25 @@
     EOS
   end
 
+  it 'prints a warning when a helper file exits early' do
+    write_file_formatted "helper_with_exit.rb", "exit 999"
+
+    expect {
+      run_command "--require ./helper_with_exit.rb"
+    }.to raise_error(SystemExit)
+    output = normalize_durations(last_cmd_stdout)
+    expect(output).to eq unindent(<<-EOS)
+
+      While loading ./helper_with_exit.rb an `exit` / `raise SystemExit` occurred, RSpec will now quit.
+      Failure/Error: exit 999
+
+      SystemExit:
+        exit
+      # ./helper_with_exit.rb:1:in `exit'
+      # ./helper_with_exit.rb:1#{spec_line_suffix}
```

Comment:
This output needs to be adapted on JRuby as it comes out differently (from the build):
```diff
Diff:

       @@ -1,9 +1,8 @@
        
        While loading ./helper_with_exit.rb an `exit` / `raise SystemExit` occurred, RSpec will now quit.
       -Failure/Error: exit 999
       +Failure/Error: Unable to find org/jruby/RubyKernel.java to read failed line
        
        SystemExit:
          exit
       -# ./helper_with_exit.rb:1:in `exit'
        # ./helper_with_exit.rb:1:in `<main>'
```
Can you adjust this spec to cope with that?

Reply:
This diff is still slightly wrong, line 106 `# ./helper_with_exit.rb:1:in `exit'` is missing entirely on Jruby too